scanomatic/data_processing/phenotyper.py

```diff
@@ -1745,7 +1745,9 @@ def _arr_tester(arr):
                 else:
                     raise ValueError("Any check failed")
             except ValueError:
-                return any(e.any() for e in arr.ravel())
+                return any(
+                    e.any() for e in arr.ravel() if isinstance(e, np.ndarray)
```

Comment:
how does this work is the item is an empty dict vs a dict with something in it? In the latter case I believe it ought to return true.